### DISCUSSION TITLE
AssetMapper compatibility ?

### DISCUSSION THREAD

**Discussion by stloc** [brefphp/bref#1741] [Support] [answered]
**cloudwatch logs** 
`
"message": "Uncaught PHP Exception Twig\\Error\\RuntimeError: \"An exception has been thrown during the rendering of a template (\"The asset mapper directory \"assets/\" does not exist.\").\" at mes-evenements.html.twig line 16",`

![image](https://github.com/brefphp/bref/assets/4832070/24383ee8-aee3-48db-b002-1fd13f7ce6f1)

in mes-evenements.html.twig :
`{% block importmap %}{{ importmap(['app', 'agenda']) }}{% endblock %}`

![image](https://github.com/brefphp/bref/assets/4832070/0ee8416e-a8f3-4fb5-9938-9e88ee357b15)

**Comment by t-richard** [comment, marked as answer]
As the error is your PHP logs, you should check if this folder is included in your deployment package (the zip file used by lambda).

This folder is probably excluded in `package.patterns` in your `serverless.yml` file.

The config shown in your screenshot is only used to upload the given directory to S3 so that the assets are available to be downloaded by a browser.

**Comment by stloc** [comment]
thank you @t-richard 
indeed that's it but I thought /assets wasn't necessary? only /public/assets
